content/CORE/UIReference/JailsPluginsVMs/Jails/JailsScreens.md

```diff
@@ -10,7 +10,7 @@ The Jails screen displays a list of jails installed on your system. Use to add,
 
 ![JailsScreen](/images/CORE/Jails/JailsScreen.png "Jails Screen")
 
-Use the blue **Columns** dropdown list to display options to change the information displayed in the lis to of tables. Options are Select **All**, **JID**, **Boot**, **State**, **Release**, **IPv4**, **IPv6**, **Type**, **Template**, **Basejail** or **Reset to Defaults**.
+Use the blue **Columns** dropdown list to display options to change the information displayed in the list of tables. Options are Select **All**, **JID**, **Boot**, **State**, **Release**, **IPv4**, **IPv6**, **Type**, **Template**, **Basejail** or **Reset to Defaults**.
```

Comment:
why is select capitalized? Is it part of the UI element with each of the other options following this? suggest not putting them in bold and caps if you are not typing exactly as on screen. If select is not, then just lc

Reply:
looks like it should have been 
```suggestion
Use the blue **Columns** dropdown list to display options to change the information displayed in the list of tables. Options are **Select All**, **JID**, **Boot**, **State**, **Release**, **IPv4**, **IPv6**, **Type**, **Template**, **Basejail** or **Reset to Defaults**.
```